src/js/components/RangeSelector/doc.js

```diff
@@ -72,7 +72,27 @@ export const doc = RangeSelector => {
 };
 
 export const themeDoc = {
-  'global.colors.controls': {
+  'global.colors.focus': {
+    description: 'TThe color when there is a focus',
+    type: 'string',
+    defaultValue: 'accent-1',
+  },
+  'global.edgeSize': {
+    description: 'The possible sizes for the margin, padding and gap',
+    type: 'object',
+    defaultValue: 'small',
+  },
+  'global.borderSize': {
+    description: 'The size of the border',
+    type: 'string',
+    defaultValue: 'undefined',
```

Comment:
I think there should be a defaultValue for this one.  can you please double check?